onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -2769,12 +2772,14 @@ Status TensorrtExecutionProvider::CreateNodeComputeInfoFromGraph(const GraphView
     }
   }
 
+  // Generate cache suffix in case user would like to customize cache prefix
+  std::string cache_suffix = "_" + GetCacheSuffix(fused_node.Name(), trt_node_name_with_precision);
```

Comment:
It's better to compose the cache_suffix only if user specifies cache_prefix.
You can move this line down to where it check "!cache_prefix_.empty()"

Reply:
Thanks! This has been applied